deployment/packer/template.js

```diff
@@ -48,6 +48,7 @@
                 "Service": "Tiler",
                 "Environment": "{{user `stack_type`}}"
             },
+            "subnet_id": "subnet-3781f46e",
```

Comment:
Since we hard coded this, it might be nice to have a comment noting such to avoid confusion for new devs in the future.